master/internal/config/config.go

```diff
@@ -88,7 +88,7 @@ type Config struct {
 	ClusterName           string                            `json:"cluster_name"`
 	Logging               model.LoggingConfig               `json:"logging"`
 	HPImportance          hpimportance.HPImportanceConfig   `json:"hyperparameter_importance"`
-
+	EnablePrometheus      bool                              `json:"enable_prometheus"`
```

Comment:
can we put this under an `observability` subsection or something (i imagine in due time we could have more related configs and I dont want to pollute the root namespace)?

Reply:
done